### DISCUSSION TITLE
How to do a github education

### DISCUSSION THREAD

**Discussion by kazantsev-v** [community/community#54529] [GitHub Education]

**Comment by ryanlua** [comment]
Hi @KVITwIm,

You can join GitHub Education by either as a student or teacher from Global Campus.

You can sign up to Global Campus at the link below: 

https://education.github.com/pack/join

**Comment by samus-aran** [comment]
Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and I'll be happy to assist!
